### PR TITLE
Solved the empty drop-down selection error

### PR DESCRIPTION
**Pull request by diegora-ms**
Fixed an issue with drop-down parameters where, when no value was selected, the Kusto query using that parameter would throw a parsing error:

image

The solution was to cast the value of the parameter to a dynamic type. This consists of changing the reference in the query:

~~~~kusto
ACSCallSummary
| where OsVersion in ({os_dropdown})
~~~~

To a dynamic cast of the drop-down menu selections:

~~~~kusto
ACSCallSummary
| where OsVersion in (dynamic([{os_dropdown}]))
~~~~

## PR Checklist

* [X] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [X] post a screenshot of templates and/or gallery changes
* [X] ensure your template has a corresponding gallery entry in the gallery folder
* [X] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [X] ensure all steps have meaningful names
* [X] ensure all parameters and grid columns have display names set so they can be localized
* [X] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [X] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [X] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [X] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Communication Services/Overview/Overview.workbook [hunk 1]
```diff
@@ -365,13 +365,7 @@
                           "durationMs": 86400000
                         },
                         "id": "2666fe6a-842f-46ad-9f75-5e52e8517ff8",
-                        "value": [
```
Comment (vincentg) [inline Workbooks/Communication Services/Overview/Overview.workbook:368]: Qq: Why this change ?

**Review by vincentg — COMMENTED** [review]
Quick question on the list being empty now

File: Workbooks/Communication Services/Overview/Overview.workbook (hunk 1 above)
Comment (diegora-ms) [reply Workbooks/Communication Services/Overview/Overview.workbook]: Sorry, that came from the graphical editor, I've restored the default values for this parameter.

**Review by vincentg — APPROVED** [review]
Looks good, thanks!
